### PR TITLE
fin servlet 

### PR DESCRIPTION
**Pull request by roroross**
finished servlet step of tute

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -28,5 +28,10 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
     response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("Hello Rosanna!");
+    //the code below shows the same thing. with p tags and without p tags show the  
+    //same font and size
```
Comment (sbland-g) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Pick one; leave the other out. It's generally considered bad practice to leave in commented out code.

**Review by sbland-g — CHANGES_REQUESTED** [review]

File: walkthroughs/week-3-server/examples/page-view-counter/src/main/java/com/google/sps/servlets/PageViewServlet.java [hunk 2]
```diff
@@ -21,15 +21,27 @@
...
+// @webservlet is an annotation that tells our server which UTL this ervlet maps to
+//when client requests the pageviews URL, this servlet is triggered
+//@WebServlet("/content")
+
+@WebServlet("/page-views") 
 public class PageViewServlet extends HttpServlet {
 
   private int pageViews = 0;
-
+  //do get function below runs whenever a client sends a GET request to the servlet URL
```
Comment (Techget) [inline walkthroughs/week-3-server/examples/page-view-counter/src/main/java/com/google/sps/servlets/PageViewServlet.java:29]: These 2 sentences are a bit duplicated

File: walkthroughs/week-3-server/examples/page-view-counter/src/main/java/com/google/sps/servlets/PageViewServlet.java [hunk 3]
```diff
@@ -21,15 +21,27 @@
...
+  //aka browser seds a GET request whenever I visit a URL
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     pageViews++;
+    //^ since the doget function runs everytime the pageviews URL is requested, 
+    //this means the pageviews variable tracks how many times the page has been viewed
 
+    /*
```
Comment (Techget) [inline walkthroughs/week-3-server/examples/page-view-counter/src/main/java/com/google/sps/servlets/PageViewServlet.java:35]: You many want to remove this comment? As the following snippet is quite similar.

File: walkthroughs/week-3-server/examples/page-view-counter/src/main/webapp/index.html [hunk 4]
```diff
@@ -7,5 +7,6 @@
   <body>
     <h1>Intro to Servlets</h1>
     <p>Click <a href="/page-views">here</a> to view content generated by a servlet.</p>
+    <!-- <p>Click <a href="/content">here</a> to view afkdjfka;jdkjajfadjfa; generated by a servlet.</p> -->
```
Comment (Techget) [inline walkthroughs/week-3-server/examples/page-view-counter/src/main/webapp/index.html]: Similiar as above, do you need the comment?

**Review by roroross — COMMENTED** [review]
reflected comments and feedback in latest git push

**Pull request closed without merging**